charts/pac-quota-controller/values.yaml

```diff
@@ -104,13 +104,11 @@ events:
 
 metrics:
   enable: true
-  port: 8443
-  certPath: /tmp/k8s-metrics-server/metrics-certs
 
 prometheus:
   enable: false
   serviceMonitor:
-    enable: true
+    enable: false
```

Comment:
Isn't using ServiceMonitor the best practice when using prometheus?
I am ok with setting true here. (keess is defaulting to true)